pages/team-assignments.tsx

```diff
@@ -1,75 +1,20 @@
-import st from 'components/Tabs/Tabs.module.scss';
-import { useState } from 'react';
 import DashboardWrapper from 'components/Dashboard/DashboardWrapper';
-import Tab from 'components/SubmissionsTable/Tab';
-import Card from 'components/MashCards/MashCard';
 import { GetServerSideProps } from 'next';
 import { isAuthorised } from 'utils/auth';
-import { MashReferral } from 'types';
 import { getAllMashReferrals } from 'lib/mashReferral';
+import MashDashboard from 'components/MashDashboard/MashDashboard';
+import { MashReferral } from 'types';
 
 interface Props {
-  referrals: MashReferral[];
+  referrals: [];
+  mashReferral: MashReferral[];
 }
 
 export const TeamAssignments = ({ referrals }: Props): React.ReactElement => {
```

Comment:
we can drop this export, we already have a default export at the bottom of the file